.travis/template.txt

```diff
@@ -7,6 +7,8 @@ matrix:
  include:
   PACKAGES_MATRIX
 install: 
+  - echo "$PWD"
+  - if [ -n "$TRAVIS_PULL_REQUEST" ] && [ "$PACKAGE" != CHECK ]; then  DO_IGNORE=FALSE;  for ARG in $(echo "$PACKAGE");do . $PWD/.travis/test_package.sh "$PWD" "$ARG";  echo "DO_IGNORE is $DO_IGNORE";  if [ "$DO_IGNORE" = "TRUE" ]; then  travis_terminate 0; fi; done;fi
```

Comment:
That logic is wrong, in `.travis.yml`. See [here for a PR modifying Triangulation_2: Triangulation_3 can be ignored, but the full test of "Triangulation_2 Triangulation_3 Union_find" is ignored!][1]

[1]: https://travis-ci.org/CGAL/cgal/jobs/380773569